book/src/2.getting-started/3.running-in-teams.md

```diff
@@ -1,54 +1,88 @@
+Now that your agent is running locally, let's deploy it to Microsoft Teams for testing. This guide will walk you through the process.
 
-## Installing Teams Toolkit
+## Teams Toolkit
 
-First, you'll need to install the Teams Toolkit extension:
+Teams Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment.
 
-1. Visit the [Teams Toolkit installation guide](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode)
-2. Follow the instructions to install for your preferred IDE
+### Install Teams Toolkit (TTK)
 
-> [!TIP]
-> The Teams Toolkit extension makes it easy to debug and test your Teams applications directly in Microsoft Teams. If you'd like to learn the various things that it helps automate, check out [this doc](../4.teams/1.core-concepts.md)
+First, you'll need to install the Teams Toolkit IDE extension:
 
-## Adding Teams Configuration
+1. Visit the [Teams Toolkit installation guide](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode).
+2. Follow the instructions to install for your preferred IDE.
 
-Your agent needs some basic configuration to work with Teams. Add this by running:
+> [!NOTE]<br/>
+> The Teams Toolkit extension makes it easy to debug and test your Teams applications directly in Microsoft Teams. If you'd like to learn what it helps automate, check out [Teams core concepts](../4.teams/1.core-concepts.md)
+
+## Adding Teams configuration files via `teams` CLI
+
+To configure your agent for Teams, run the following command in the terminal inside your quote-agent folder:
 
 ```bash
+# (if you have `teams` CLI installed globally, use `teams` instead of `npx`)
 npx @microsoft/teams.cli config add ttk.basic
 ```
 
-This command adds the necessary configuration files that Teams Toolkit expects, including:
+> [!TIP] <br/>
+> The `ttk.basic` configuration is a basic setup for Teams Toolkit. It includes the necessary files and configuration to get started with Teams development.<br/>
+> Explore more advanced configurations as needed with teams config --help.<br>
 
-- Environment setup
-- Teams app manifest template
-- Debug configurations
+This [CLI](./7.developer-tools/1.cli/README.md) command adds configuration files required by Teams Toolkit, including:
 
-## Debugging in Teams
+- Environment setup in the `env` folder and root `.env` file
+- Teams app manifest in the `appPackage` folder (if not already present)
+- Debug instructions in `.vscode/launch.json` and `.vscode/tasks.json`
+- TTK automation files to your project (e.g. `teamsapp.local.yml`)
 
-Once you've installed Teams Toolkit and added the configuration:
+> [!NOTE]<br/>
```

Comment:
Hm, does this matter? (is it worth including here)

Reply:
I'm anticipating people being confused by the two cli's and how they have some overlapping functionality but not 1:1. I'll keep this for now but we can discuss further if you'd like.